django/applications/catmaid/control/similarity.py

```diff
@@ -132,22 +134,22 @@ def serialize_similarity(similarity, with_scoring=False, with_objects=False):
 
 def serialize_pointcloud(pointcloud, with_locations=False, with_images=False):
     data = {
-        'id': similarity.id,
-        'user_id': similarity.user_id,
-        'creation_time': similarity.creation_time,
-        'project_id': similarity.project_id,
-        'config_id': similarity.config_id,
-        'name': similarity.name,
-        'description': similarity.description,
-        'source_path': similarity.source_path,
+        'id': pointcloud.id,
```

Comment:
likely what was meant

Reply:
That's true, but actually I think this whole function slipped in accidentally and should be removed. `serialize_pointcloud` is defined and used in `control/pointcloud.py`. The `similarity` module shouldn't need to care about pointcloud serialization.